README.md

```
@@ -94,14 +94,31 @@ excalidraw-cli convert diagram.excalidraw --format svg --no-export-background
 | `[Label @fillStyle:hachure @backgroundColor:#a5d8ff]` | Styled node | Add inline node style attributes |
 | `->` | Arrow | Connection |
 | `-->` | Dashed Arrow | Dashed connection |
-| `-> "text" ->` | Labeled Arrow | Connection with label |
+| `-> "text" ->` | Labeled Arrow | Connection with a double-quoted label |
+| `-> 'text' ->` | Labeled Arrow | Connection with a single-quoted label |
 
 ### Example DSL
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)